src/pages/CreateShop.js

```diff
@@ -0,0 +1,171 @@
+import React, {useState, useEffect, useRef} from 'react';
+import "bootstrap/dist/css/bootstrap.min.css";
+import {Form, Button, Container, Row} from 'react-bootstrap'
+import "./CreateShop.css";
+import axios from 'axios';
+import {AiOutlineCloudUpload} from "react-icons/ai";
+import Error from "../components/Error";
+import Loading from "../components/Loading";
+import {Navigate, useNavigate } from "react-router-dom";
+import {GoogleMap, useLoadScript, Marker} from "@react-google-maps/api";
+import usePlacesAutocomplete, {getGeocode, getLatLng} from 'use-places-autocomplete';
+import {Combobox, ComboboxInput, ComboboxPopover, ComboboxList, ComboboxOption} from "@reach/combobox";
+import "@reach/combobox/styles.css";
+
+const CreateShop = () => {
+    const shopsURL = "http://127.0.0.1:8000/shops/";
```

Comment:
Can you use the ServerHostnameContext to determine which hostname for the URL (an example to reference is like 12-13 in Home.js)